nix_update/__init__.py

```diff
@@ -56,6 +58,12 @@ def parse_args(args: list[str]) -> Options:
         help="Regex to extract version with, i.e. 'jq-(.*)'",
         default="(.*)",
     )
+    parser.add_argument(
+        "-vk",
```

Comment:
Not sure we need a short-hand for that:

```suggestion
```

I see this more being used in scripts where the long-form is more descriptive.